stdlib/3/tkinter/__init__.pyi

```diff
@@ -274,31 +284,44 @@ class Misc:
     def nametowidget(self, name): ...
     register: Any
     def configure(self, cnf: Optional[Any] = ..., **kw): ...
-    config: Any
+    config = configure
     def cget(self, key): ...
     __getitem__: Any
     def __setitem__(self, key, value): ...
     def keys(self): ...
-    def pack_propagate(self, flag=...): ...
-    propagate: Any
-    def pack_slaves(self): ...
-    slaves: Any
-    def place_slaves(self): ...
-    def grid_anchor(self, anchor: Optional[Any] = ...): ...
-    anchor: Any
+    @overload
+    def pack_propagate(self, flag: bool) -> Optional[bool]: ...
+    @overload
+    def pack_propagate(self, flag: List[Literal["_noarg_"]] = ...) -> None: ...
```

Comment:
I initially typed it as `List[Literal["noarg"]]` because stubtest didn't like leaving out the type of flag. I have now reverted that and whitelisted these propagate methods.